aries-site/src/examples/templates/persistent-navigation/MinimalSidebarExample.js

```diff
@@ -24,6 +25,7 @@ import {
   Splits,
   StatusInfoSmall,
 } from 'grommet-icons';
+import { ThemeContext } from 'styled-components';
```

Comment:
Why using styled-components context instead of Grommet's?

Reply:
Thanks, I've updated to use grommet themecontext and confirmed functionality persists.